### ISSUE TITLE
about GRPO

### ISSUE THREAD

**Issue by KkHu-Kistch** [alibaba/ROLL#14]
Regarding the GRPO algorithm, have you guys considered UnsLoth's EfficientGRPO method for saving GPU memory? Its principle is similar to Liger kernel's fused_linear_cross_entropy. This method directly passes hidden_states and lm_head to avoid generating large logits tensors with vocabulary dimension.

https://github.com/unslothai/unsloth-zoo/blob/main/unsloth_zoo/rl_replacements.py#L158

https://github.com/linkedin/Liger-Kernel/blob/main/src/liger_kernel/ops/fused_linear_cross_entropy.py

**Comment by PanAndy** [comment]
Thank you for your interest in ROLL.
Thank you for your valuable suggestions regarding the implementation of EfficientGRPO; we'll take a look at it. 
We also welcome contributions to ROLL~
